### PR TITLE
*/*: disable static-libs

### PR DESCRIPTION
**Pull request by jsmolic**
@SoapGentoo

### CODE REVIEW

File: dev-libs/shhopt/shhopt-1.1.7-r3.ebuild [hunk 1]
```diff
@@ -16,7 +16,7 @@ KEYWORDS="amd64 ppc x86"
 PATCHES=( "${FILESDIR}"/${P}-build.patch )
 
 src_compile() {
-	emake CC=$(tc-getCC)
+	emake CC=$(tc-getCC) AR=$(tc-getAR)
```
Comment (SoapGentoo) [inline dev-libs/shhopt/shhopt-1.1.7-r3.ebuild:19]: :heart:

File: net-libs/libident/libident-0.32-r1.ebuild [hunk 2]
```diff
@@ -10,12 +12,19 @@ SRC_URI="http://people.via.ecp.fr/~rem/libident/${P}.tar.bz2"
 LICENSE="public-domain"
 SLOT="0"
 KEYWORDS="~alpha amd64 ~arm arm64 ~hppa ~ia64 ~mips ppc ppc64 ~s390 ~sparc x86"
-IUSE=""
 
-RDEPEND=""
-DEPEND="${RDEPEND}"
+DOCS=( AUTHORS ChangeLog NEWS README )
```
Comment (SoapGentoo) [inline net-libs/libident/libident-0.32-r1.ebuild:16]: why did you add this one? I'm assuming it installs something like `README.BeOS`

File: net-libs/openmq-cclient/openmq-cclient-4.4.1.ebuild [hunk 3]
```diff
@@ -73,6 +73,11 @@ src_prepare() {
 	eautoreconf
 }
 
+src_configure() {
+	econf --disable-static
+}
+
 src_install() {
 	emake install DESTDIR="${D}"
```
Comment (SoapGentoo) [inline net-libs/openmq-cclient/openmq-cclient-4.4.1.ebuild:81]: replace by `default` and bring some docs in?

File: sys-fabric/libmlx5/libmlx5-1.0.1.ebuild [hunk 4]
```diff
@@ -12,11 +12,22 @@ inherit openib
...
-RDEPEND="
-		!sys-fabric/openib-userspace"
+DEPEND="sys-fabric/libibverbs:${SLOT}"
+RDEPEND="!sys-fabric/openib-userspace"
 block_other_ofed_versions
+
+src_prepare() {
+	default
+
+	sed -i -e '/CFLAGS=\"$CFLAGS -Werror\"/d' configure
```
Comment (SoapGentoo) [inline sys-fabric/libmlx5/libmlx5-1.0.1.ebuild:23]: missing `|| die`

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: net-libs/libident/libident-0.32-r1.ebuild (hunk 2 above)
Comment (jsmolic) [reply net-libs/libident/libident-0.32-r1.ebuild:16]: To drop `dodoc` from `src_install`, I can return `dodoc` if you wish

File: net-libs/openmq-cclient/openmq-cclient-4.4.1.ebuild [hunk 5]
```diff
@@ -73,6 +73,12 @@ src_prepare() {
...
 }
 
+src_configure() {
+	econf --disable-static
+}
+
 src_install() {
-	emake install DESTDIR="${D}"
+	default
+	dodoc "${WORKDIR}"/mq/src/doc/en/*
```
Comment (SoapGentoo) [inline net-libs/openmq-cclient/openmq-cclient-4.4.1.ebuild:82]: `dodoc -r "${WORKDIR}"/mq/src/doc/en/.`
always try to avoid globs if possible

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: net-libs/openmq-cclient/openmq-cclient-4.4.1.ebuild (hunk 5 above)
Comment (jsmolic) [reply net-libs/openmq-cclient/openmq-cclient-4.4.1.ebuild]: Fixed, thanks
